content/06.methods.md

```diff
@@ -113,6 +113,12 @@ Note that GATK v4.0.12.0 was used for all of these steps except VariantFiltratio
 This single-sample workflow is available in the [D3b GitHub repository](https://github.com/d3b-center/OpenPBTA-workflows).
 References can be obtained from the [Broad Genome References on AWS](https://s3.console.aws.amazon.com/s3/buckets/broad-references/hg38/v0/) bucket, and a general description of references can be found at https://s3.amazonaws.com/broad-references/broad-references-readme.html.
 
+##### Pathogenicity calling
+
+We annotated germline VCFs (gVCFs) with SnpEff [@pmid:22728672], followed by ANNOVAR [@doi:10.1093/nar/gkq603] and InterVar [@doi:10.1016/j.ajhg.2017.01.004].
+We removed variants with a minor allele frequency of >= 0.1% in ExAC [@doi:10.1093/nar/gkw971] and any gnomAD population [@doi:10.1038/s41586-020-2308-7].
```

Comment:
I'm confused by this sentence. What were the variants removed from? I assume they are still present in the VCF, but was a pathogenic call removed? I think it would also be helpful to have a brief justification for this sentence. Why does appearance in gnomAD justify removal?

Also, what kind of annotations were added in the sentence before? I assume from context that it is pathogenicity, but we should say that explicitly.

Reply:
updated in latest commit - see what you think